src/server/session.ts

```diff
@@ -669,7 +669,7 @@ namespace ts.server {
             const occurrences = project.getLanguageService().getOccurrencesAtPosition(file, position);
 
             if (!occurrences) {
-                return emptyArray;
+                return undefined;

```

Comment:
In `master`, whichever is more consistent with the other requests.  My impression is that it's `undefined`.  In `release-2.5`, whichever it used to return (also `undefined`?).